src/content/docs/more-integrations/open-source-telemetry-integrations/jenkins/monitoring-jenkins-ot.mdx

```diff
@@ -1,12 +1,14 @@
 ---
-title: Monitoring Jenkins with OpenTelemetry and New Relic
+title: Monitor Jenkins with OpenTelemetry and New Relic 
 tags:
   - Integrations
   - Open source telemetry integrations
   - OpenTelemetry
   - Jenkins
   - Quickstart
 metaDescription: Here is a simple example to setup the Jenkins OpenTelemetry plugin to send data to New Relic.
+redirects:
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/jenkins/monitoring-jenkins-ot
 ---
 
 import opentelemetryJenkis01 from 'images/opentelemetry_screenshot-full_jenkis-01.png'
```

Comment:
i know you didn't do this but if you felt like fixing. a minor typo in there for image: "opentelemetryJenkis01"

Reply:
@zuluecho9 I fixed that typo for this Jenkins image.